tuxemon/db.py

```diff
@@ -106,9 +106,10 @@ class JSONMonster(JSONMonsterOptionalFields):
     weight: float
     sprites: JSONMonsterSprites
 
-class JSONStat(TypedDict):
-    value: int 
-    dividing: bool
+class JSONStat(TypedDict, total=False):
+    minvalue: int
```

Comment:
I would prefer `value` and `max_deviation` instead, as it is more clear in the fixed damage case, and it can be generalized to the normal distribution it necessary.

Reply:
fixed